src/gradualizer.erl

```diff
@@ -1,13 +1,15 @@
 %%% @doc Main external API of the Gradualizer
 %%%
 %%% The functions `type_check(file|module|dir)' accept the following options:
+%%% - `{i, Dir}': Include path for `-include' and `-include_lib' when checking
+%%%   Erlang source files. Specify multiple times for multiple include paths.
 %%% - `stop_on_first_error': if `true' stop type checking at the first error,
 %%%   if `false' continue checking all functions in the given file and all files
 %%%   in the given directory.
 %%% - `print_file': if `true' prefix error printouts with the file name the
 %%%   error is from.
-%%% - `crash_on_error': if `true` crash on the first produced error
-%%% - `return_errors': if `true`, turns off error printing and errors
+%%% - `crash_on_error': if `true' crash on the first produced error
```

Comment:
use matching ticks, either ' or `

Reply:
this is good old edoc notation for verbatim http://erlang.org/doc/apps/edoc/chapter.html#verbatim-quoting

Reply:
That's super weird, but okay

Reply:
That's super weird, but okay

Reply:
Yeah, it's weird but probably older than markdown. If we'd want to generate HTML docs from this, we could publish it on github pages or on Hex.pm if we make a hex package. If we don't, there is no need to use edoc syntax.